pkg/types/types.go

```diff
@@ -522,6 +522,18 @@ type ConfigMapReloader struct {
 	Disabled bool   `yaml:"disabled,omitempty"`
 }
 
+type Elasticsearch struct {
+	Version        string                     `yaml:"version"`
+	Mem            string                     `yaml:"mem,omitempty"`
+	Replicas       int                        `yaml:"replicas,omitempty"`
+	Persistence    *ElasticsearchPersistence  `yaml:"persistence,omitempty"`
```

Comment:
I think we can abstract this to a common Persistence for both Elastic and prometheus?

Reply:
Will change, thanks